build.py

```diff
@@ -415,7 +415,7 @@ def extra_configs(dot_config, kbuild_output):
     # Patterns for matching kernel images by architecture
     if arch == 'arm':
         patterns = ['zImage', 'xipImage']
-    elif arch == 'arm64':
+    elif arch == 'arm64' or arch == "riscv":
```

Comment:
There's also `Image.gz` which is actually defined as the default one in `arch/riscv/Makefile`:
```
KBUILD_IMAGE    := $(boot)/Image.gz
```
I'm not sure why that is the case though, and maybe `Image` is more suitable to boot in practice.

Reply:
@khilman In the new build.py there's a similar thing using a dictionary and I initially put `Image.gz` for `riscv`:
https://github.com/kernelci/kernelci-core/pull/46/files#diff-d9df6c8ff856deb49f5362dc3c59e305R53

so the kernel image uploaded ends up being `Image.gz`:
https://staging.kernelci.org/build/id/5cc73390cf3a0f0df35f958d/

If this is no good then I can change it to `Image`, or have both as the first one in the list will take priority.

Reply:
@gctucker please add Image as the first choice, as this works better with the current upstream u-boot, until they add booti and compressed image support.

I'll update/rebase this patch to list both as well.

Reply:
OK I've changed `kernelci/build.py` to have this:
```python
# Hard-coded binary kernel image names for each CPU architecture
KERNEL_IMAGE_NAMES = {
    'arm': ['zImage', 'xipImage'],
    'arm64': ['Image'],
    'arc': ['uImage.gz'],
    'i386': ['bzImage'],
    'mips': ['uImage.gz', 'vmlinux.gz.itb'],
    'riscv': ['Image', 'Image.gz'],
    'x86_64': ['bzImage'],
    'x86': ['bzImage'],
}
```